testdata/cheats/Parse.t.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0;
+
+import "forge-std/Test.sol";
```

Comment:
I'd do this without forge-std and remove forge-std as a submodule

Reply:
what are the reasons we use `ds-test` in cheatcode tests and not `forge-std`? Forge one has more `assertEq`, that's why I used it. For example, `ds-test` doesn't have `assertEq(bool, bool)` and you need to do `assertEq(bool ? 1 : 0, bool ? 1 : 0)` to make it work.

Reply:
i think it may worth it to switch to `ds-test` in cheatcode tests in another PR